### PR TITLE
Fix #11 bulk logging and make buffers faster

### LINKED ISSUE #11: Fully log data
Current branch does not log observations/actions/rewards for _every_ step. We should be logging all this information, if not to wandb, to a file. This likely means some configuration in `CustomCallbacks`.

### PR DESCRIPTION
**Pull request by Aphoh**

### CODE REVIEW

**Comment by Aphoh** [comment]
Working in this run here: https://wandb.ai/social-game-rl/energy-demand-response-game/runs/2zylku6v/overview?workspace=user-aphoh

File: gym-socialgame/gym_socialgame/envs/buffers.py [hunk 1]
```diff
@@ -79,64 +79,65 @@ def entropy(self):
...
         self.buffer_size += 1
+        if self.buffer_size >= self.buffer.shape[0]:
+            self.buffer = np.append(self.buffer, np.zeros(self.buffer.shape), axis=0)
 
     def get_params(self):
         #import pdb; pdb.set_trace()
-        means = np.mean(self.buffer, axis=0)
-        stds = np.std(self.buffer, axis=0)
+        means = np.mean(self.buffer[:self.buffer_size], axis=0)
+        stds = np.std(self.buffer[:self.buffer_size], axis=0)
```
Comment (tsgoten) [inline gym-socialgame/gym_socialgame/envs/buffers.py:97]: Why do we make a copy of the buffer? Especially since we're just calculating means and stds?

**Review by tsgoten — COMMENTED** [review]
Looks good. I'll start a run with this branch.

**Review by tsgoten — APPROVED** [review]
Forgot to approve.

**Pull request merged**